OBSDOCS-460: Clean up about page and old references (#66606)
openshift/openshift-docs: snippets/logging-kibana-dep-snip.adoc

// Text snippet included in the following assemblies:
//
// logging/cluster-logging.adoc
//
// Text snippet included in the following modules:
//
//

:_content-type: SNIPPET

[NOTE]
====

> The Kibana web console is now deprecated and will be removed in a future logging release.

Reviewer: ```suggestion
The Kibana web console is deprecated and is planned to be removed in a future logging release.
```
